tmk_core/protocol/chibios/main.c

```diff
@@ -101,6 +101,39 @@ void midi_ep_task(void);
 //   }
 // }
 
+/* Early initialisation
+ */
+__attribute__((weak)) void early_hardware_init_pre(void) {
+#if FALSE && defined(STM32_BOOTLOADER_ADDRESS) // remove the FALSE when we've migrated all existing __early_init invocations
```

Comment:
Is there a better option that FALSE here so someone can prep for the eventual refactor? (also name are difficult...)
```suggestion
#if defined(ENABLE_STM32_BOOTLOADER_ADDRESS) && defined(STM32_BOOTLOADER_ADDRESS) // remove the FALSE when we've migrated all existing __early_init invocations
```

Reply:
I'll end up doing the refactoring at some point after this regardless, specifically for the dual-bank flash version that requires the extra GPIO. Will leave this name for now and potentially refactor as required at the time.